src/middlewares/confirmationMiddleware.js

```diff
@@ -0,0 +1,39 @@
+import response from '../utils/response';
+import messages from '../utils/messages';
+import models from '../models';
+
+const { Request, User } = models;
+
+/**
+ * ReConfirm Travel Request Middleware
+ * @param {Object} req - Express request object
+ * @param {Oject} res - Express response object
+ * @param {Function} next - Express next middleware function
+ * @returns {Object} - Returns Object
+ */
+const requestConfirmation = async (req, res, next) => {
+  const { requestId } = req.params;
+  const { confirmation } = req.query;
+  try {
+    if (confirmation === undefined || confirmation === 'false') {
```

Comment:
LET USE JOIN HERE

Reply:
I don't think Joi is needed on it. I did a little update on it, please review